swap_meet/errors.py

```diff
@@ -0,0 +1,10 @@
+
+class InvalidIDError(Exception):
```

Comment:
👍 Nice creation of your own error type. It provides a nice representation of the invalid id that was supplied.

However, at the end of the day, the actual issue is that a value of the wrong type is being passed, so a `TypeError `might fit this more precisely. True, it wouldn't have a dedicated field for the invalid id, but we can pass any arbitrary to the `TypeError` constructor, and that can be accessed through its `.args` property.

I feel like `InvalidIDError` would be more apprpriate if we did receive an int, but there were some int ranges that weren't valid (maybe negative?). But in that case, we might argue that `ValueError` is descriptive enough.